openapi.yaml

```diff
@@ -0,0 +1,418 @@
+      tags:
+        - "Person"
+      summary: "Delete a person"
+      description: "Delete a person with same ID"
+      operationId: "deletePerson"
+      parameters:
+        - name: "id"
+          in: "path"
+          description: "Person`s ID to delete"
+          required: true
+          schema:
+            type: "string"
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                type: object
+                items:
+                  $ref: "#/components/schemas/PersonResponse"
+        "400":
+          description: "Invalid ID supplied"
+        "404":
+          description: "Person not found"
+
+  /person/{idPerson}/passport:
+    get:
+      tags:
+        - "Person"
+      summary: "Find person by passport ID"
+      description: "Get current active passports of a person "
+      operationId: "getPasswordByPersonIdAndPassportId"
+      parameters:
+        - name: "idPerson"
+          in: "path"
+          description: "Person`s ID to get"
+          required: true
+          schema:
+            type: "string"
+        - name: "active"
+          in: "query"
+          description: "Status of passport to get"
+          required: true
+          schema:
+            type: "boolean"
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PassportResponse"
+        "400":
+          description: "Invalid passport number supplied"
+        "404":
+          description: "Passport not found"
+
+
+  /passport:
+    get:
+      tags:
+        - "PassportStore"
+      summary: "Find passport with necessary date range"
+      description: "Return a list with passports with date range"
+      operationId: "getPassports"
+      parameters:
+        - name: "dateStart"
+          in: "query"
+          description: "Start date of range"
+          required: true
+          schema:
+            type: "string"
+            format: "date"
+
+        - name: "dateEnd"
+          in: "query"
+          description: "End date of range"
+          required: true
+          schema:
+            type: "string"
+            format: "date"
+      responses:
+        "200":
+          description: "Successful operation"
+          content:
+            '*/*':
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PassportResponse"
+        "400":
+          description: "Invalid Order"
+
+    post:
+      tags:
+        - "PassportStore"
+      summary: "Add a passport"
+      description: "Add a  passport"
+      operationId: "addPassport"
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/PassportRequest'
```

Comment:
it fixed also all endpoints are moved on /person/{id} base because it means that person is in the personal account of the application/ It is some security moment for starting working you need to open your account